### PR TITLE
Register studiodev.is-a-good.dev

### PR DESCRIPTION
**Pull request by ashfaq1994**
Added `studiodev.is-a-good.dev` using the [CLI](https://www.npmjs.com/package/@is-a-good-dev/cli).

### CODE REVIEW

File: sub-logs/studiodev.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "username": "ashfaq1994",
+        "email": "tricksfree.in@gmail.com"
+    },
+
+    "target": {
+        "A": {
+            "name": "studiodev.is-a-good.dev",
+            "value": ["103.191.209.47"]
+        },
+         "CNAME": {
```
Comment (Tweak4141) [inline sub-logs/studiodev.json:12]: You can only have an A record or CNAME.

File: sub-logs/studiodev.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "owner": {
+        "username": "ashfaq1994",
+        "email": "tricksfree.in@gmail.com"
+    },
+
+    "target": {
+        "A": {
+            "name": "studiodev.is-a-good.dev",
```
Comment (Tweak4141) [inline sub-logs/studiodev.json:9]: name should only be studiodev

File: sub-logs/studiodev.json (hunk 1 above)
Comment (Tweak4141) [inline sub-logs/studiodev.json:12]: Invalid CNAME.

**Review by Tweak4141 — CHANGES_REQUESTED** [review]

File: sub-logs/studiodev.json (hunk 2 above)
Comment (ashfaq1994) [reply sub-logs/studiodev.json]: Update the package.json, Please check and let me know.

File: sub-logs/studiodev.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        "email": "tricksfree.in@gmail.com"
+    },
+
+    "target": {
+        "A": {
+            "name": "studiodev",
+            "value": ["103.191.209.47"]
+        },
+        "A": {
+            "name": "www",
```
Comment (Tweak4141) [inline sub-logs/studiodev.json:13]: A record is invalid, we do not support nested domains. You will have to do without www.

**Review by Tweak4141 — CHANGES_REQUESTED** [review]

**Review by Tweak4141 — APPROVED** [review]

**Comment by ashfaq1994** [comment]
WHy PR not merged ?

**Pull request closed without merging**